src/QuartzNET-DynamoDB/DynamoBootstrapper.cs

```diff
@@ -414,6 +422,29 @@ private void CreateSchedulerTable(IAmazonDynamoDB client)
             client.CreateTable(createRequest);
 
             EnsureTableActive(client, createRequest.TableName);
+
+            SetTimeToLive(client, DynamoConfiguration.SchedulerTableName, "ExpiresUtcEpoch");
+        }
+
+        /// <summary>
+        /// Enables the time to live feature for the provided table on the given attribute name.
+        /// Note that it may take up to one hour for the change to fully process.
```

Comment:
Yeah, that is from the amazon docs. Interesting, I figured leave it be rather than poll and block until it finishes!